Submodels/file_handling.py

```
@@ -137,6 +147,9 @@ def __init__(self,
                      params_CB: dict,
                      params_duct: dict,
                      case_name: str,
+                     ref_length: float,
+                     external_input : bool = False,
+                     domain_boundaries : Optional[list[float, float, float, float]] = [None, None, None, None],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)